pandas/_libs/ops.pyi

```diff
@@ -0,0 +1,43 @@
+from typing import (
+    Any,
+    Callable,
+)
+
+import numpy as np
+
+binop = Callable[[Any, Any], Any]
+bool_op = Callable[[Any, Any], bool]
```

Comment:
from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#conventions

> Type variables and aliases you introduce purely for legibility reasons should be prefixed with an underscore to make it obvious to the reader they are not part of the stubbed API.

we normally capitalize as well